pkgs/development/interpreters/python/hooks/pytest-check-hook.sh

```diff
@@ -37,6 +37,7 @@ function pytestCheckPhase() {
         disabledTestsString=$(_pytestComputeDisabledTestsString "${disabledTests[@]}")
       args+=" -k \""$disabledTestsString"\""
     fi
+    disabledTestFiles=($disabledTestFiles)
```

Comment:
I think we need to add a comment here explaining the situation.

Reply:
It seems like probably we shouldn't actually merge this,  but instead take one of the approaches @jtojnar suggested.